### DISCUSSION TITLE
App Crashed After Adding Custom Parameters to Ecommerce Events

### DISCUSSION THREAD

**Discussion by kennnnnnnis** [invertase/react-native-firebase#5971] [Q&A]
I am relatively new to Firebase and React Native and I am trying to add custom parameters such as "product sizes" to ecommerce events (e.g. purchase). But seems like the Firebase library only allows self-determined parameters as the app crashes immediately after doing so. May I know if it is a known issue?

What I would like to achieve is to set up custom parameters such as "membership types" such that I can use to filter my data in the analytics section. Thanks!

**Comment by mikehardy** [comment]
This is probably best asked on stackoverflow - I'm unfamiliar with it myself, but that's a semi-official support channel for the firebase crew

**Reply by kennnnnnnis** [reply]
Thanks for your suggestions! I will raise this on Stackoverflow as well.
